leobot_web_server/scripts/web_server.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env nodejs
+
+const static = require('node-static');
+
+const fs = require('fs');
+const path = require('path');
+const rosnodejs = require('rosnodejs');
+
+rosnodejs.initNode('leobot_web_server').then(() => {
+    const nh = rosnodejs.nh;
+
+    console.log('rosnodejs handle: ' + nh);
+
+    nh.getParam('port').then((p) => {
```

Comment:
please check this example how parameters are read https://github.com/RethinkRobotics-opensource/rosnodejs/blob/kinetic-devel/src/examples/standard.js#L44-L46